src/isomorphic/children/__tests__/ReactChildren-test.js

```diff
@@ -850,4 +855,45 @@ describe('ReactChildren', () => {
         'to render a collection of children, use an array instead.',
     );
   });
+
+  if (ReactDOMFeatureFlags.useFiber) {
+    describe('with fragments enabled', () => {
+      beforeEach(() => {
+        ReactFeatureFlags = require('ReactFeatureFlags');
+        ReactFeatureFlags.disableNewFiberFeatures = false;
+      });
+
+      it('warns for keys for arrays of elements in a fragment', () => {
+        spyOn(console, 'error');
+        class ComponentReturningArray extends React.Component {
+          render() {
+            return [<div />, <div />];
+          }
+        }
+
+        ReactTestUtils.renderIntoDocument(
+          React.createElement(ComponentReturningArray),
+        );
+
+        expectDev(console.error.calls.count()).toBe(1);
+        expectDev(console.error.calls.argsFor(0)[0]).toContain(
```

Comment:
Do you mind changing this to a `toBe` to make sure we have the full error message captured? You might need a `normalizeCodeLocInfo` (copy it from another file) if it contains the file and line.